deps.bzl

```diff
@@ -355,6 +355,20 @@ def install_buildbuddy_dependencies():
 	version = "v0.50.0",
     )
 
+    go_repository(
+        name = "com_github_aws_aws_sdk_go",
+        importpath = "github.com/aws/aws-sdk-go",
+	    version = "v1.30.14",
+	    sum = "h1:vZfX2b/fknc9wKcytbLWykM7in5k6dbQ8iHTJDUP1Ng="
```

Comment:
Looks like some extra spaces slipped in before "version" and "sum". Should look like:


    go_repository(
        name = "com_github_aws_aws_sdk_go",
        importpath = "github.com/aws/aws-sdk-go",
        version = "v1.30.14",
        sum = "h1:vZfX2b/fknc9wKcytbLWykM7in5k6dbQ8iHTJDUP1Ng="
    )

Reply:
tabs vs. spaces =)